doc/arkode/guide/source/Butcher.rst

```diff
@@ -1109,18 +1109,15 @@ embedding are A-stable; additionally the method is L-stable (from :cite:p:`KenCa
 
 .. _Butcher.ARK_7_3_4_I:
 
-ARK-7-3-4 (implicit)
-^^^^^^^^^^^^^^^^^^^^^^^
+ARK437L2SA-DIRK-7-3-4
+^^^^^^^^^^^^^^^^^^^^^
 
-.. index:: ARK-7-3-4 ESDIRK method
+.. index:: ARK437L2SA-DIRK-7-3-4 method
 
 Accessible via the constant ``ARKODE_ARK437L2SA_DIRK_7_3_4`` to
 :c:func:`ARKStepSetTableNum()` or
 :c:func:`ARKodeButcherTable_LoadDIRK()`.  This is the implicit portion
-of the 4th order additive method from :cite:p:`KenCarp:19`.
-
-..
```

Comment:
Should the comment below about stability be included?

Reply:
I had commented those out before because I thought that I had added the statement by mistake; instead of "hiding" it in the now-public documentation source, I chose to remove it entirely.